packages/core/src/Table/stories/TableSamples/PropsTableSection.tsx

```diff
@@ -0,0 +1,117 @@
+import { useState, useMemo } from "react";
+import range from "lodash/range";
+import {
+  HvTableContainer,
+  HvTable,
+  HvTableBody,
+  HvTableHead,
+  HvTableHeader,
+  HvTableRow,
+  HvTableCell,
+  HvTypography,
+  HvActionsGeneric,
+  useHvPagination,
+  useHvRowSelection,
+  useHvBulkActions,
+  useHvData,
+} from "@hitachivantara/uikit-react-core";
+
+import { HvTableSection } from "packages/core/src/TableSection";
```

Comment:
Just for consistency, we should import `HvTableSection` from `@hitachivantara/uikit-react-core` in the stories.